src/components/Card/Card.module.css

```diff
@@ -28,14 +28,13 @@
 .front,
 .back {
   width: 100%;
-  border: 1px solid rgb(85, 85, 85);
   backface-visibility: hidden;
   position: absolute;
   top: 0;
   left: 0;
   height: 100%;
   overflow: hidden;
-  border-radius: calc(var(--card-width) * 0.1);
+
```

Comment:
just remove empty line. This should not happen after you start using fresh `main` branch